README.md

```diff
@@ -10,11 +10,11 @@ Typescript wrapper for [supranational/blst](https://github.com/supranational/bls
 
 | OS / Arch     | binary name | Node                               |
 | ------------- | ----------- | ---------------------------------- |
-| Linux / x64   | linux-x64   | 14, 16, 17, 18, 20 |
-| Linux / arm64 | linux-arm64 | 14, 16, 17, 18, 20 |
-| Windows / x64 | win32-x64   | 14, 16, 17, 18, 20 |
-| OSX / x64     | darwin-x64  | 14, 16, 17, 18, 20 |
-| OSX / M1      | -\*         | -\*                                |
+| Linux / x64   | linux-x64   | 16, 17, 18, 20 |
+| Linux / arm64 | linux-arm64 | 14, 16, 18, 20 |
+| Windows / x64 | win32-x64   | 16, 17, 18, 20 |
+| OSX / x64     | darwin-x64  | 16, 17, 18, 20 |
+| OSX / M1      | darwin-arm64  | 16, 17, 18, 20 |
```

Comment:
Nit: The term OSX is not used anymore. I think it's better to say macOS instead.

Nit2: I am in favour of saying arm64 instead of M1. There are a lot of new chips from Apple Silicon since M1.

Reply:
:+1: